src/controllers/controller.bar.js

```diff
@@ -86,6 +86,10 @@ defaults._set('horizontalBar', {
 	}
 });
 
+function getValue(scale, dataPoint) {
+	return typeof dataPoint === 'string' ? Number(dataPoint) : scale.getRightValue(dataPoint);
```

Comment:
I'm not sure how to do that. If I add the following to `scale.linearbase.js` then all the tests start failing:

		getRightValue: function() {
			return Chart.Scale.prototype.getRightValue.call(this);
		},